oembed/Oembed.php

```diff
@@ -103,13 +103,14 @@ protected static function matches_scheme($url, $scheme) {
 	 */
 	protected static function autodiscover_from_url($url)
 	{
-		// Fetch the URL (cache for a week by default)
-		$service = new RestfulService($url, 60 * 60 * 24 * 7);
-		$body = $service->request();
-		if (!$body || $body->isError()) {
+		$service = new GuzzleHttp\Client();
+
+		$request = $service->request('GET', $url);
+		// Fetch from Oembed URL (cache for a week by default)
+		if(!$request || $request->getStatusCode() !== 200) {
```

Comment:
Same as previous response mainly.
Forgot to take it out because more focused on getting it done than focus on everything around it due to hackday.

Caching is by the way hardly ever of any use, unless the embedded item is embedded a dozen times a minute via the CMS.